site/app/templates/admin/admin_gradeable/AdminGradeableCreate.twig

```diff
@@ -533,6 +533,26 @@
         //
         // Gradeable Type Radio Button Events
         //
+        
+        $("input[type='radio']").change(validate);
+        
+        function validate() 
+        {
+            var sub1 = $('input[name=type]');
+            var sub2 = $('input[electronic_gradeable_presets]');
+            //var sub3 = $('input[vcs]');
+            //var sub4 = $('input[scanned_exam]');
+            
+            if ( $(sub1).is(':checked') || $(sub2).is(':checked')) 
+            {
+                $(".btn#submit1").removeAttr("disabled", falsse);
```

Comment:
There's a syntax error on this line, I think you meant `false` instead

Reply:
@shailpatels I resolved this syntax error, Please check the changes again.